frontend/web/src/features/bookingengine/BookingOverview.js

```diff
@@ -16,7 +16,7 @@ import People from "@mui/icons-material/PeopleAltOutlined";
 import Back from "@mui/icons-material/KeyboardBackspace";
 import publicKeys from "../../utils/const/publicKeys.json";
 
-const stripePromise = loadStripe(publicKeys.STRIPE_PUBLIC_KEYS.LIVE);
+const stripePromise = loadStripe(publicKeys.STRIPE_PUBLIC_KEYS.TEST);
```

Comment:
Zou je dit naar de live modus terug kunnen brengen?

```suggestion
const stripePromise = loadStripe(publicKeys.STRIPE_PUBLIC_KEYS.LIVE);
```